wiki/Ranking_Criteria/en.md

```diff
@@ -31,22 +31,17 @@ Glossary
 
 ### Metadata
 
+- **Official sources**: Places directly from the artist or their label such as official websites, CD Scans, or official uploads to websites like Bandcamp, YouTube, or SoundCloud. Third party websites such as wikis, databases (vndb, vgmdb, etc.), or music services (Spotify, iTunes, Amazon, etc.) do not count as official sources.
 - **vs.**: versus
-- **&**: and
-- **feat./ft.**: featuring
+- **feat.**: featuring
 - **CV**: character voice, usually used to refer to the voice actor of a fictional character.
-- **Lenticular brackets**: Also known as Sumitsukikakko and written as `【】`.
-- **Corner brackets**: Also known as Kagikakko and written as `「」`.
-- **Square brackets**: `[]`
-- **asterisk**: `*`
-- **umlauts**: `ü`, `ö`, `ä` and `ß`
-- **TV Size**: A song where the particular version is used in a television program, web series, or direct to video series, such as an opening, ending, or insert song. Indie/doujin series are not included in this.
-
-Note: If a beatmapset's song contains matching sections to the TV size song, in the same order, and is roughly the same length as the official TV Size song, the beatmapset's edit will also count as TV Size
-
+- **Asterisk**: `*`
+- **Umlauts**: `ü`, `ö`, `ä` and `ß`
+- **Whitespace**: A visual spacing between characters, not always a literal space. If a character already has space in it, such as all of Japanese, that counts as whitespace.
```

Comment:
what does this comment about japanese mean (are you referring to punctuation?)

Reply:
full width characters that include white space

Reply:
should clarify that then, because "all of Japanese" does not include whitespace, lol